troposphere/static/js/components/modals/unsupported/modernizrTest.js

```diff
@@ -0,0 +1,69 @@
+define(function (require) {
```

Comment:
Just wanted to mention that we no longer need to use the admin style wrapping. This style was originally needed to support require.js